docs/pages/versions/unversioned/sdk/splash-screen.mdx

```diff
@@ -80,6 +85,116 @@ export default function App() {
+      description: 'Resize mode to render the splash screen image on a device, must be one of "cover", "contain" or "native".',
+    },
+    {
+      name: '<platform>.image',
+      description:
+        'Image to render on the splash screen, must be a png file. Both local paths or remote URLs are supported.',
+    },
+    {
+      name: '<platform>.dark.*',
+      description: 'Dark mode specific configuration for the splash screen, defined as nested object, with the same properties without the ".dark" prefix.'
+    },
+    {
+      name: 'ios.tabletImage',
+      platform: 'ios',
+      description:
+        'Image to render on the splash screen on tablets, must be a png file. Both local paths or remote URLs are supported.',
+    },
+    {
+      name: 'android.mdpi',
+      platform: 'android',
+      description: 'The natural sized image, or baseline, of the splash screen image, must be a .png file.',
+    },
+    {
+      name: 'android.hdpi',
+      platform: 'android',
+      description: 'The 1.5x scale of the sized image of the splash screen image, must be a .png file.',
+    },
+    {
+      name: 'android.xhdpi',
+      platform: 'android',
+      description: 'The 2x scale of the sized image of the splash screen image, must be a .png file.',
+    },
+    {
+      name: 'android.xxhdpi',
+      platform: 'android',
+      description: 'The 3x scale of the sized image of the splash screen image, must be a .png file.',
+    },
+    {
+      name: 'android.xxxhdpi',
+      platform: 'android',
+      description: 'The 4x scale of the sized image of the splash screen image, must be a .png file.',
+    },
+  ]}
+/>
+
+Here is an example of using the config plugin in the Expo config file:
+
+```json app.json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-splash-screen",
+        {
+          "android": {
+            "resizeMode": "contain",
+            "backgroundColor": "#FFFFFF",
+            "image": "./assets/splash-android.png",
+            "dark": {
+              "backgroundColor": "#000000",
+              "image": "./assets/splash-android-dark.png"
+            }
+          },
+          "ios": {
+            "resizeMode": "contain",
+            "backgroundColor": "#FFFFFF",
+            "image": "./assets/splash-ios.png",
+            "dark": {
+              "backgroundColor": "#000000",
+              "image": "./assets/splash-ios-dark.png"
+            }
+          }
+        }
+      ]
+    ]
+  }
+}
+```
+
+<ConfigClassic>
+
+Learn how to configure notifications with the app manifest `splash` properties.
+
+- [`splash`](../config/app.mdx#splash)
+- [`android.splash`](../config/app.mdx#splash-2)
+- [`ios.splash`](../config/app.mdx#splash-1)
+
+</ConfigClassic>
+
+<ConfigReactNative>
+
+Learn how to configure the native projects in the [installation instructions in the `expo-splash-screen` repository](https://github.com/expo/expo/tree/main/packages/expo-splash-screen#-installation-in-managed-expo-projects).
```

Comment:
```suggestion
See how to configure the native projects in the [installation instructions in the `expo-splash-screen` repository](https://github.com/expo/expo/tree/main/packages/expo-splash-screen#-installation-in-bare-react-native-projects).
```

I think the link is supposed to redirect the user to the bare React Native section?